includes/managers/controls.php

```diff
@@ -798,6 +801,15 @@ public function add_control_to_stack( Controls_Stack $element, $control_id, $con
 			'index' => null,
 		];
 
+		$control_type = 'controls';
+		if (
+			Performance::is_optimized_control_loading_feature_enabled()
+			&& Performance::should_optimize_controls()
+			&& $this->is_style_control( $control_data )
+		) {
+			$control_type = 'style_controls';
+		}
```

Comment:
doesn't it break 3rd parties that expect the style controls to be under the `controls` key?